lwt/cohttp_lwt.ml

```diff
@@ -298,7 +298,14 @@ module Make_server(IO:IO)
     Filename.concat docroot rel_path
 
   let respond ?headers ?(flush=true) ~status ~body () =
-    let encoding = Cohttp_lwt_body.transfer_encoding body in
```

Comment:
It's probably better to test the expected behaviour of `respond` isnt it?  With and without user-specified headers and a few different body formats.